<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInputDialogArticleAllowance</class>
 <widget class="QDialog" name="CInputDialogArticleAllowance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lava2res.qrc">
    <normaloff>:/images/res/icon.ico</normaloff>:/images/res/icon.ico</iconset>
  </property>
  <widget class="QComboBox" name="comboBoxMask">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Artikelbezeichnung</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.Artikelnummer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.Artikelnummer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hersteller</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Warengruppe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Standort</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBoxAuto">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMask">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Artikel:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetArticle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>881</width>
     <height>351</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Bestand / bestellt / max.Kap.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Artikelbezeichnung</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Artikelnummer(n)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hersteller</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Warengruppe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Standort</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMaskSet">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>setzen</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>881</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>150</red>
         <green>147</green>
         <blue>140</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxNotCloseTheDialog">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>dieses Fenster zum schnelleren hinzufügen geöffnet halten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>hinzufügen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Fenster schliessen</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>comboBoxMask</tabstop>
  <tabstop>lineEditMask</tabstop>
  <tabstop>pushButtonMaskSet</tabstop>
  <tabstop>checkBoxAuto</tabstop>
  <tabstop>checkBoxNotCloseTheDialog</tabstop>
  <tabstop>tableWidgetArticle</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonOk</tabstop>
 </tabstops>
 <resources>
  <include location="lava2res.qrc"/>
 </resources>
 <connections/>
</ui>
